kubernetes/annotations_and_labels.md

```diff
@@ -5,10 +5,11 @@
 - `app` - value should contain the name of the application. Should be applied
   to every Kubernetes resource associated with an application. This should be
   also used as replicas selector. E.g. `app=kvm-operator`.
-- `giantswarm.io/cluster-id` - value should contain guest cluster ID which this
-  object is part of. E.g. `giantswarm.io/cluster-id=eggs2`.
-- `giantswarm.io/customer-id` - value should contain guest cluster's customer
-  ID. `giantswarm.io/customer-id=track-hunter`.
+- `giantswarm.io/cluster` - value should contain guest cluster ID which this
+  object is part of. E.g. `giantswarm.io/cluster=eggs2`.
+- `giantswarm.io/organization` - value should contain guest cluster's
+organization ID as displayed in the front-end
+`giantswarm.io/organization=track-hunter`.
```

Comment:
Is organization a drop in replacement for customer? We have multiple organizations per customer in happa. Or I confuse things?

Reply:
Yes it's a replacement.

AFAIK the customer naming is from the previous microservices platform. It doesn't make sense now since an installation belongs to a single customer.

Reply:
Thanks for clarification!